src/main/java/frc/constants/field/Field.java

```diff
@@ -1,17 +1,126 @@
 package frc.constants.field;
 
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.wpilibj.DriverStation;
+import frc.constants.field.enums.*;
 import frc.utils.DriverStationUtils;
+import frc.utils.math.FieldMath;
 
 public class Field {
 
 	public static final DriverStation.Alliance RELATIVE_FIELD_CONVENTION_ALLIANCE = DriverStation.Alliance.Blue;
 
+	public static final double LENGTH_METERS = 17.548225;
+	public static final double WIDTH_METERS = 8.0518;
```

Comment:
```suggestion
	public static final double FIELD_LENGTH_METERS = 17.548225;
	public static final double FIELD_WIDTH_METERS = 8.0518;
```
I think that that way it would be more clear, but it is comprletely up to you

Reply:
its in class FIELD its clear